ci/cron/daily-compat.yml

```diff
@@ -457,6 +470,7 @@ jobs:
             MSG="Daily tests passed: $COMMIT_LINK"
             REPORT='```
         speedy_perf: $(speedy_perf)
+        $(oracle_perf_results)
```

Comment:
Should you add a label here to specify that what follows is the performance of the JSON API on Oracle?

Reply:
Label should not be necessary here as we are generating descriptive labelled stats from the job itself, we can do a sample run and see how it prints out, @garyverhaegen-da can you kick that off please ?